hack/annotator/annotator.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# Script to automate the task of adding or removing the "exclude" annotation from/to all Elastic resources in a Kubernetes cluster.
+
+set -euo pipefail
+
+ANN_KEY=${ANN_KEY:-"common.k8s.elastic.co/pause"}
```

Comment:
Let's use the latest annotation `eck.k8s.elastic.co/managed`?

Reply:
The reason I went with the old annotation is because the documentation is targeted at people who are trying to upgrade from 1.0.0-beta1. The new annotation is only available from 1.1.0 onwards.